hashgraph-transfer.md

```diff
@@ -13,33 +13,80 @@ When transferring ownership of a repository from [Hashgraph](https://github.com/
 
 ## PRE-TRANSFER Checks
 
-Prior to the day of transfer the following items shall be defined and made available to the **Repository Transfer Agent**
+- Prior to the day of transfer the following items shall be defined and made available to the **Repository Transfer Agent**
+  - Target Repository Name (like hiero-sdk-foo)
+  - Maintainer Group Name (like hiero-sdk-foo-maintainer)
+  - Maintainers List (should be 2-3 people)
+  - Committer Group Name (like hiero-sdk-foo-commiter)
+  - Committers List
 
-- Target Repository Name (like hiero-sdk-foo)
-- Maintainer Group Name (like hiero-sdk-foo-maintainer)
-- Maintainers List (should be 2-3 people)
-- Committer Group Name (like hiero-sdk-foo-commiter)
-- Committers List 
+- Point of contact for questions during transfer is identified and provided to the **Repository Transfer Agent**
+
+- Impending Change has been communicated
+  - Internal consumers are aware that the repository is moving to [Hiero-Ledger][hiero]
+    - Prework has been performed on an internal branch to change references from **hashgraph** to **hiero**
+  - External consumers are aware that the repository is moving to [Hiero-Ledger][hiero]
+ 
+- If publishing artifacts to maven central:
+  - Appropriate service account is configured in [Hiero-Ledger][hiero]
+  - Maven Central namespace is available for publishing
+  - List of currrently published packages (from [Hashgraph][hashgraph]) is provided to **Repository Transfer Agent**
+- If publishing gradle plugins to gradle plugin central:
+  - Appropriate service account is configured in [Hiero-Ledger][hiero]
+  - List of currrently published packages (from [Hashgraph][hashgraph]) is provided to **Repository Transfer Agent**
+- If publishing to npmjs:
+  - Appropriate service account is configured in [Hiero-Ledger][hiero]
+  - List of currrently published packages (from [Hashgraph][hashgraph]) is provided to **Repository Transfer Agent**
+- If codecov artifacts are captured for the repository
+  - **Repository Transfer Agent** is made aware
+
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository variables and variable states and provide to
+  the TA prior to day of transfer
+- The [@hashgraph/devops-ci team][devops-ci] will capture the repository secrets and provide this list to the TA prior
+  to day of transfer
+
+## Day Before Checks
+
+The day before a repository is transferred the TA and AA shall verify the following checks have been complete
+
+- DCO Audit is complete
+- License Audit is complete
+- Pre-Transfer Checks are complete
+  - TA has the following:
+    - New repository Information
+    - List of MC packages
```

Comment:
I would spell out `Maven Central` here instead of `MC`

Reply:
```suggestion
    - List of Maven Central packages
 ```